clients/csharp-uwp/UWPVoiceAssistantSample/DialogResponseQueue.cs

```diff
@@ -77,7 +77,7 @@ public void TryDequeue()
                     if (nextResponse.MessageMedia != null)
                     {
                         this.operationInProgress = true;
-                        this.outputAdapter.EnqueueDialogAudio(nextResponse.MessageMedia);
+                        _ = this.outputAdapter.PlayAudioAsync(nextResponse.MessageMedia);
```

Comment:
Since PlayAudioAsync can be awaited and ends when outputended fires, we can make this async and simplify this class significantly. I can do that in a follow-up PR though.